When x11-libs/wxGTK is built with the odbc use flag, scorched3d fails to build.
bug #285384 - patch from Bob Johnson

diff -urN scorched/src/common/porting/windows.h fix_scorched/src/common/porting/windows.h
--- scorched/src/common/porting/windows.h	2009-09-21 10:21:16.000000000 -0700
+++ fix_scorched/src/common/porting/windows.h	2009-09-21 10:39:02.000000000 -0700
@@ -11,8 +11,8 @@
 
 typedef char * 		LPSTR;
 typedef const char *	LPCTSTR;
-typedef unsigned int 	DWORD;
-typedef unsigned int *  LPDWORD;
+typedef unsigned long 	DWORD;
+typedef DWORD *         LPDWORD;
 typedef unsigned char * LPBYTE;
 typedef int 		INT;
 typedef long         	LONG;
